deptofdefense/solo
# Bug update celery connection string to use escaped postgres host

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist
I have…
~~[ ] attached screenshots illustrating relevant behavior before and after my changes.~~
~~[ ] review and update SSAG documentation if needed.~~

## Summary of Changes
This pull request…
- Fix bug in using escaped postgresql hostname in celery broker url settings

## Testing
To verify the changes proposed in this pull request…
1. run environment locally
2. ensure celery worker can connect to postgresql for message queing

## Screenshots
N/A
